apps/minifront/src/state/ibc.ts

```diff
@@ -193,16 +193,18 @@ export const ibcValidationErrors = (state: AllSlices) => {
   return {
     recipientErr: !state.ibc.destinationChainAddress
       ? false
-      : !validateAddress(state.ibc.chain, state.ibc.destinationChainAddress),
+      : !validateUnknownAddress(state.ibc.chain, state.ibc.destinationChainAddress),
     amountErr: !state.ibc.selection
       ? false
       : amountMoreThanBalance(state.ibc.selection, state.ibc.amount),
   };
 };
 
-const validateAddress = (chain: Chain | undefined, address: string): boolean => {
+const validateUnknownAddress = (chain: Chain | undefined, address: string): boolean => {
   if (!chain || address === '') return false;
-  return bech32IsValid(address, chain.addressPrefix);
+  const { prefix, words } =
+    bech32.decodeUnsafe(address, Infinity) ?? bech32m.decodeUnsafe(address, Infinity) ?? {};
+  return !words || prefix !== chain.addressPrefix;
```

Comment:
This feels more appropriately placed in the bech32 package. Also, what's the deal with `Infinity`? Some code comments may help here.

Reply:
our bech32m package may be consumed to validate penumbra addresses.

this is validating an address from an unknown chain in an unknown format.

other formats are the responsibility of the application.

Reply:
added comment explaining